vspace/src/main/java/edu/asu/diging/vspace/web/staff/ListImagesController.java

```diff
@@ -48,7 +50,19 @@ public String listSpaces(@PathVariable(required = false) String page,
         model.addAttribute("totalPages", imageService.getTotalPages());
         model.addAttribute("currentPageNumber", pageNo);
         model.addAttribute("totalImageCount", imageService.getTotalImageCount());
-        List<IVSImage> images = imageService.getImages(pageNo, sortedBy, order);
+        List<IVSImage> images;
+        if(imageCategory.equals("Show All")) {
```

Comment:
should not be hard-coded; especially since it's used before put in enum or constant in an interface